src/utils/index.ts

```diff
@@ -23,13 +24,14 @@ export const scrollTo = (opts?: ScrollToOptions) => {
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const parseLogData = (value?: any, columnName?: string) => {
 	const dateColumnNames = ['date', 'datetime', 'time', 'timestamp', 'p_timestamp'];
+	const stringifiedValue = _.toString(value);
```

Comment:
I dont think this will be a good idea. ie converting all the values to string. 
If I use this parseLogData function anywhere else, all the values will be string, lets say I am doing export or showing a json on the frontend.